mods/tuxemon/maps/leather_town.tmx

```diff
@@ -206,8 +206,8 @@
   </object>
   <object id="90" name="Sign: Route 3" type="event" x="544" y="16" width="16" height="16">
    <properties>
-    <property name="act10" value="translated_dialog spyder_route3_sign"/>
-    <property name="act20" value="translated_dialog leather_town_sign_leather_route"/>
+    <property name="act10" value="translated_dialog alert_route"/>
```

Comment:
Why did you change the name to alert_route?

Reply:
I had another idea. I'm going to change it back